earthfile2llb/earthfile2llb.go

```diff
@@ -97,6 +99,16 @@ func Earthfile2LLB(ctx context.Context, target domain.Target, opt ConvertOpt) (m
 	if err != nil {
 		return nil, errors.Wrapf(err, "resolve build context for target %s", target.String())
 	}
+
+	ftrs, err := features.GetFeatures(bc.Earthfile.Version)
+	if err != nil {
+		return nil, errors.Wrapf(err, "resolve build context for target %s", target.String())
+	}
+
+	if ftrs.ReferencedSaveOnly {
+		fmt.Printf("TODO feature-flip referenced save artifact as local feature in a future PR.\n")
```

Comment:
Somewhat unrelated, but this will apply to both save artifact and save image, right?

Reply:
That was my intention, a single bool to flip both of those on and off.